app/src/test/java/rick_and_morty/ui/characters/CharactersViewModelTest.kt

```diff
@@ -56,7 +57,9 @@ class CharactersViewModelTest {
         onBlocking { it.getCharacters(1) } doReturn (characterResultsDto)
     }
 
-    private val classToTest by lazy { CharactersViewModel(characterRepository) }
+    private var eventBus: EventBus = mock()
```

Comment:
The test for posting the navigation event is missing.

Reply:
Done